r2dbc/boot-jooq-r2dbc-sample/src/main/resources/db/migration/postgresql/V1__01_init.sql

```diff
@@ -5,9 +5,9 @@ CREATE
     TABLE
         posts(
             id uuid NOT NULL DEFAULT uuid_generate_v4(),
-            title text,
-            content text,
-            status text,
+            title text NOT NULL,
```

Comment:
@coderabbitai, how to handle NOT BLANK scenario? if use input value as `""` it is allowing so I want to not allow it.